back-end/e-learning/app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+  function login(Request $request)
+  {
+    $user= User::where('email', $request->email)->first();
```

Comment:
observe correct spacing here

```suggestion
    $user = User::where('email', $request->email)->first();
```